### DISCUSSION TITLE
Pepe gets behind auras

### DISCUSSION THREAD

**Discussion by jfdhuiz** [Repooc/ElvUI_DynamicStatusIcons#15] [Q&A] [answered]
Hey, Pepe gets behind auras. So I put Pepe to bottom instead of top, but now he's under my castbar. I want to see Pepe in his full glory, every time. How can I achieve this?

**Comment by Repooc** [comment]
There is no option as I felt auras and castbar elements were more important.  I will look at possibly adding an option to adjust the level so u can make it be on top then.

**Comment by Repooc** [comment, marked as answer]
Just added frame level in 1.07, and 1.08 will have frame strata but didnt think i needed it until someone tested with another addon and it needed it lol.
